app/controllers/likes_controller.rb

```diff
@@ -6,9 +6,11 @@ def create
       post_id: @post.id
     )
     if new_like.save
+      flash[:success] = 'Post saved successfully'
       redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", notice: 'Success!'
     else
       redirect_to "/users/#{@post.author_id}/posts/#{@post.id}", alert: 'Error occured!'
+      flash.now[:error] = 'Post not saved'
```

Comment:
- [ ] The notice should be `Like saved successfully` instead of `Post ...`. On the line no. 9 and 13 both.